operations/customer-success/README.md

```diff
@@ -1,5 +1,27 @@
 # Customer Success
 
+## Vision
```

Comment:
@emilyhollinger, I changed the header level on this. Please let me know if it looks as expected, otherwise happy to revert.

Reply:
@justinegeffen That works for me.  Wondering if we change the header from "Vision" to "Purpose?" I'm 0/5 on that

Reply:
That would work well I think.